src/Controller/Traits/DocumentTreeConfigTrait.php

```diff
@@ -1,146 +0,0 @@
-<?php
-declare(strict_types=1);
-
-/**
- * Pimcore
- *
- * This source file is available under two different licenses:
- * - GNU General Public License version 3 (GPLv3)
- * - Pimcore Commercial License (PCL)
- * Full copyright and license information is available in
- * LICENSE.md which is distributed with this source code.
- *
- *  @copyright  Copyright (c) Pimcore GmbH (http://www.pimcore.org)
- *  @license    http://www.pimcore.org/license     GPLv3 and PCL
- */
-
-namespace Pimcore\Bundle\AdminBundle\Controller\Traits;
-
-use Pimcore\Bundle\AdminBundle\Event\ElementAdminStyleEvent;
-use Pimcore\Config;
-use Pimcore\Model\Document;
-use Pimcore\Model\Element\ElementInterface;
-use Pimcore\Model\Site;
-use Pimcore\Tool\Admin;
-use Pimcore\Tool\Frontend;
-
-/**
- * @internal
- */
-trait DocumentTreeConfigTrait
```

Comment:
this needs to be removed as well https://github.com/pimcore/pimcore/blob/11.x/bundles/SeoBundle/src/Controller/Traits/DocumentTreeConfigWrapperTrait.php

Reply:
or we keep the `DocumentTreeConfigTrait` as deprecated and use it internally until core 11.1 is released. Otherwise you can't use `pimcore/pimcore:11.0.x` with `admin-ui-classic-bundle:11.1`. wdyt?